src/classes/application.js

```diff
@@ -12,22 +13,35 @@ module.exports = class Application extends Base {
     const config = require(path.join(process.cwd(), 'config/environment.js'))(process.env.NODE_ENV)
     const MiddlewareRouter = require(path.join(process.cwd(), 'app/middleware.js'))
     const Router = require(path.join(process.cwd(), 'app/router.js'))
+    const log = new Log()
 
+    log.trace('Ash server creating express app instance')
     const app = express()
     _app.set(this, app)
 
-    const log = new Log()
-
-    log.info('Ash server loading middleware')
+    log.trace('Ash server loading middleware')
     const middlewareDir = path.join(process.cwd(), 'app/middleware')
     loadMiddleware(MiddlewareRouter.definition, app, middlewareDir)
 
-    log.info('Ash server loading routes')
+    log.trace('Ash server loading routes')
     const options = {routesDir: path.join(process.cwd(), 'app/routes')}
     app.use(createRoutes(Router.definition, options))
 
+    const initializerDir = path.join(process.cwd(), 'app/initializers')
+    if (fs.existsSync(initializerDir)) {
+      const initializers = fs.readdirSync(initializerDir)
```

Comment:
Does `fs.readdirSync` guarantee the order of the files it reads in? I would assume it's alphabetical. Using Ash, I would hope that the initialization steps happen in a reliable order.

10/10